### PR TITLE
Change Bytes read

### PR DESCRIPTION
**Pull request by izavala**
Working on getting second failing test to pass. Should have this done soon, will set the tcp port to try and connect after the connection has closed and check to make sure an error is signaled as the port should refuse a connection.

### CODE REVIEW

**Comment by izavala** [comment]
Test are passing in Visual Studio but seem to be failing both the Linux and Windows, need to look into why both test are failing here

**Comment by izavala** [comment]
Looked through the logs and it looks like the test below is now failing, need to look into why it's failing: 

https://github.com/petabridge/akkadotnet-healthcheck/blob/64b56a003f2967c1f23d02394640ae4e4fd7b9c1/src/Akka.HealthCheck.Tests/AkkaHealthCheckIntegrationSpec.cs#L44-L84

**Comment by izavala** [comment]
When we are testing to see if we are still connected to the TCP port, we are not attempting to interact with the socket so it seems it may not be updating the status. 

https://github.com/petabridge/akkadotnet-healthcheck/blob/64b56a003f2967c1f23d02394640ae4e4fd7b9c1/src/Akka.HealthCheck.Tests/AkkaHealthCheckIntegrationSpec.cs#L67-L82

File: src/Akka.HealthCheck.Tests/AkkaHealthCheckIntegrationSpec.cs [hunk 1]
```diff
@@ -23,15 +24,21 @@ namespace Akka.HealthCheck.Tests
...
+            : base(GetConfig(), output: helper)
         {
         }
 
-        public static readonly Config HealthcheckConfig = @"
+
+
+        public static Config GetConfig()
+        {
+            var PortNumber = ThreadLocalRandom.Current.Next(10000, 64000);
```
Comment (Aaronontheweb) [inline src/Akka.HealthCheck.Tests/AkkaHealthCheckIntegrationSpec.cs]: LGTM

File: src/Akka.HealthCheck.Tests/AkkaHealthCheckIntegrationSpec.cs [hunk 2]
```diff
@@ -69,17 +78,21 @@ public async Task Should_load_custom_HealthCheck_system_correctly()
...
                 var bytesRead = await tcpClient.GetStream().ReadAsync(new byte[10], 0, 10);
                 bytesRead.Should().Be(0);
             }
             catch
             {
             }
-
-            tcpClient.Connected.Should().BeFalse();
+            //Second client should not be able to connect as socket has been closed
+            AwaitCondition(()=> !tcpClient2.Connected);
```
Comment (Aaronontheweb) [inline src/Akka.HealthCheck.Tests/AkkaHealthCheckIntegrationSpec.cs]: Good

File: src/Akka.HealthCheck.Tests/Transports/SocketStatusTransportSpecs.cs [hunk 3]
```diff
@@ -73,16 +80,16 @@ public async Task Should_successfully_repeatedly_open_signal()
...
             tcpClient.Connected.Should().BeTrue();
 
             // special case - need to test the NULL pattern
             var result3 = await Transport.Go(null, CancellationToken.None);
             result.Success.Should().BeTrue();
 
-            bytesRead = tcpClient.Available;
-            bytesRead.Should().Be(0);
+            
+            AwaitAssert(() => tcpClient.Available.Should().Be(8));
```
Comment (Aaronontheweb) [inline src/Akka.HealthCheck.Tests/Transports/SocketStatusTransportSpecs.cs]: @izavala shouldn't this be 0 bytes read?

Comment (izavala) [reply src/Akka.HealthCheck.Tests/Transports/SocketStatusTransportSpecs.cs]: It should read bytes as it should be getting a response, the tcpClient.Available is still active
